package.json

```diff
@@ -85,5 +85,8 @@
 		"ava": "nyc ava",
 		"test": "run-s build lint ava",
 		"precommit": "run-p test"
-	}
+	},
+	"resolutions": {
```

Comment:
@danielruf please, help me to understand more about this. Could you explain better?